src/components/Board.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './Board.css';
+import Square from './Square';
+import PropTypes from 'prop-types';
+
+
+const generateSquares = (squares, onClickCallback) => {
+  // Complete this for Wave 1
+
+}
+
+const Board = ({ squares, onClickCallback }) => {
+
+  const squareList = generateSquares(squares, onClickCallback);
+  console.log(squareList);
+  return <div className="grid" >
+    {squareList}
+  </div>
+}
+
+Board.propTypes = {
+  squares: PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.shape({
+    id: PropTypes.number.isRequired,
+    value: PropTypes.string.isRequired
+  }))),
+  onClickCallback: PropTypes.func.isRequired,
+};
```

Comment:
I'd split this out for better readability:

```suggestion
Board.propTypes = {
  squares: PropTypes.arrayOf(
    PropTypes.arrayOf(
      PropTypes.shape({
        id: PropTypes.number.isRequired,
        value: PropTypes.string.isRequired
      })
    )
  ),
  onClickCallback: PropTypes.func.isRequired,
};
```